frontend/src/components/Pages/FrontPage/AutoScheduleSection/AutoScheduleSection.tsx

```diff
@@ -166,53 +174,94 @@ const AutoScheduleList = () => {
+                <StyledDialog open={dialogOpen}>
+                    {selectedMissions.length === 1 ? (
+                        <>
                             <StyledDialog.Header>
                                 <StyledDialog.Title>
-                                    <Typography variant="h3">{TranslateText('New scheduled mission')}</Typography>
+                                    <Typography variant="h3">
+                                        {TranslateText('Update schedule of mission') + ' ' + selectedMissions[0]?.name}
+                                    </Typography>
                                 </StyledDialog.Title>
                             </StyledDialog.Header>
                             <StyledDialog.CustomContent>
-                                <SelectMissionsComponent
-                                    missions={missionDefinitions}
-                                    selectedMissions={selectedMissions}
-                                    setSelectedMissions={setSelectedMissions}
-                                    multiple={false}
-                                />
-                                {dialogOpen && selectedMissions.length === 1 && (
-                                    <StyledFormCard>
-                                        <MissionDefinitionEditDialogContent
-                                            missionDefinition={selectedMissions[0]}
-                                            fieldName="autoScheduleFrequency"
-                                            closeEditDialog={closeDialog}
-                                        />
-                                    </StyledFormCard>
-                                )}
+                                <StyledFormCard>
+                                    <AutoScheduleEditDialogContent
+                                        missionDefinition={selectedMissions[0]}
+                                        fieldName="autoScheduleFrequency"
+                                        closeEditDialog={closeDialog}
+                                    />
+                                </StyledFormCard>
+                            </StyledDialog.CustomContent>
+                        </>
+                        ) : (
+                        <>
+                            <StyledDialog.CustomContent>
+                                <StyledDialog.Header>
+                                    <StyledDialog.Title>
+                                        <Typography variant="h3">
+                                            {TranslateText('Select mission for Auto Scheduling')}
+                                        </Typography>
+                                    </StyledDialog.Title>
+                                </StyledDialog.Header>
+                                <SelectMission />
                             </StyledDialog.CustomContent>
-                        </StyledDialog>
+                        </>
```

Comment:
Remove the <></> wrapper

Reply:
Fixed - have cleaned up the wrappers here